src/main/java/com/amazonaws/encryptionsdk/AwsCrypto.java

```diff
@@ -515,6 +694,11 @@ public <K extends MasterKey<K>> CryptoResult<byte[], K> decryptData(
             SignaturePolicy.AllowEncryptAllowDecrypt,
             maxEncryptedDataKeys_);
 
+    return decryptData(cryptoHandler, ciphertext);
+  }
```

Comment:
Are these deprecations documented anywhere? 
The javadoc doesn't say anything about what should be used instead.